File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/asynchronousNotificationsofChanges/model/AlertingChangeNotification.java
package org.ngbp.jsonrpc4jtestharness.rpc.asynchronousNotificationsofChanges.model;

import org.ngbp.jsonrpc4jtestharness.rpc.receiverQueryApi.model.Alerting;

import java.util.List;

public class AlertingChangeNotification {
    public String msgType;
    private List<Alerting> alertList;
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/asynchronousNotificationsofChanges/model/AlertingChangeNotification.kt
package org.ngbp.jsonrpc4jtestharness.rpc.asynchronousNotificationsofChanges.model

import org.ngbp.jsonrpc4jtestharness.rpc.receiverQueryApi.model.Alerting

data class AlertingChangeNotification (
    var msgType: String? = null,
    var alertList: MutableList<Alerting?>? = null
)